name.abuchen.portfolio.tests/src/name/abuchen/portfolio/money/ValuesBuilder.java

```diff
@@ -0,0 +1,44 @@
+package name.abuchen.portfolio.money;
+
+import java.text.DecimalFormat;
+
+import name.abuchen.portfolio.money.Values.QuoteValues;
+
+public class ValuesBuilder
+{
+    public static <T extends Number> Values<T> createNumberValues(Values<T> orgValues)
+    {
+        return new Values<T>(orgValues.pattern(), orgValues.precision())
+        {
+            private final DecimalFormat format = new DecimalFormat(pattern());
+
+            @Override
+            public String format(Number share)
+            {
+                if (DiscreetMode.isActive())
+                    return DiscreetMode.HIDDEN_AMOUNT;
+                else
+                    return format.format(share.doubleValue() / divider());
+            }
+        };
+    }
+
+    public static <T extends Number> Values<T> createNumberValues(String pattern, int precision)
```

Comment:
I think this (test) method is not used anywhere in the code

Reply:
Correct, I've created it and then realized that the code that is involved in the parsing of monetary values didn't use the parse-methods in the Values-instance but above DecimalFormat in QUOTE_FORMAT. I've left the method in because it might come handy in future testcases where this might be needed.